porting/nimble/src/os_mbuf.c

```diff
@@ -633,19 +626,20 @@ os_mbuf_adj(struct os_mbuf *mp, int req_len)
          * after the remaining size.
          */
         len = -len;
+        if (m->om_len >= len) {
```

Comment:
If the mbuf size is greater than length than we don't need to iterate over the whole chain to count the overall length.